cumulusci/tasks/marketing_cloud/get_user_info.py

```diff
@@ -0,0 +1,42 @@
+import json
+from logging import getLogger
+
+import requests
+
+from .base import BaseMarketingCloudTask
+from .mc_constants import MC_API_VERSION
+
+
+class GetUserInfoTask(BaseMarketingCloudTask):
+    """Retrieves user info of pertaining to the currently logged in user.
+    Sanitizes and returns the payload in self.return_values."""
+
+    logger = getLogger(__name__)
+
+    def _run_task(self):
+        endpoint = f"https://{self.mc_config.tssd}.auth.marketingcloudapis.com/{MC_API_VERSION}/userinfo"
+        headers = {
+            "Authorization": f"Bearer {self.mc_config.access_token}",
+            "Content-Type": "application/json",
+        }
+        try:
+            response = requests.get(endpoint, headers=headers)
+        except requests.exceptions.HTTPError as e:
+            self.logger.error(f"Error fetching user info: {e}")
```

Comment:
Should we log here, or should we just let error handling in `cci.py` work its magic?

Reply:
I would be sure to log something here which includes response.text so we can see the error as reported by Marketing Cloud.

Also don't forget to call response.raise_for_status() to handle HTTP responses with a non-200 status.

Reply:
Actually please use the safe_json_from_response helper

Reply:
I was thinking that the try/except block would be in-place-of `raise_for_status()`, I'm happy to change it though. Are you thinking it would cover a larger swath of exception scenarios?

Reply:
requests.get doesn't raise an HTTPError when it gets a non-200 response. It returns the response object with the status that was received.